wiki/osu!supporter/en.md

```diff
@@ -85,7 +85,7 @@ osu! offers more relaxed limits on various online features to supporters:
 
 | Value | Regular limit | osu!supporter limit |
 | :-- | :-: | :-: |
-| [Pending beatmaps](/wiki/Beatmap/Category#work-in-progress-and-pending) | `4 + min(ranked beatmaps, 2)` | `8 + min(ranked beatmaps, 4)` |
+| [Pending beatmaps](/wiki/Beatmap/Category#work-in-progress-and-pending) | `2 + min(ranked beatmaps, 6)`, up to **8** | `8 + min(ranked beatmaps, 12)`, up to **20**[^pending-beatmaps-ref] |
```

Comment:
would rather put this reference in an inline comment

this is also put on the supporter limit, but the source relates to the non-supporter limit

Reply:
think it relates to both, no?

Reply:
I also like it more as a footnote, because it backs up a fact, allows you to make a reference without interfering much with the way the article is written

Reply:
> think it relates to both, no?

well if so, then it'd be weird to have it on only one of the table cells, but i guess you could have it on both.

Reply:
or as a separate empty column, like they do in tournament articles